clientd3d/d3drender_objects.h

```diff
@@ -0,0 +1,18 @@
+// Meridian 59, Copyright 1994-2024 Andrew Kirmse and Chris Kirmse.
+// All rights reserved.
+//
+// This software is distributed under a license that is described in
+// the LICENSE file that accompanies it.
+//
+// Meridian is a registered trademark.
+//
+// The render world object interface is responsible for rendering world objects 
+// (such as monsters, players, ornaments and vegetation) in the game world.
+// World objects are always drawn when the d3drenderer `draw_objects` is true.
+//
+#ifndef __D3DRENDEROBJECTS_H__
```

Comment:
My fault.. I didn't notice that d3drender.h doesn't match the style of the rest of the code.  Please use the style from other files (say, client.h) in anything you add.